editor.planx.uk/src/@planx/components/MapAndLabel/model.ts

```diff
@@ -0,0 +1,16 @@
+import { MoreInformation, parseMoreInformation } from "../shared";
+
+export interface MapAndLabel extends MoreInformation {
+  fn: string;
+  title?:string;
+  description?:string;
```

Comment:
I'd expect `title` to be required, while `description` can be optional ! Requiring titles means that components like the `SummaryList` can be guaranteed to have accurate row headers for accessibility - see context here https://github.com/theopensystemslab/planx-new/pull/2870

Also a nitpick: I personally find it much easier to read when properties are defined in the order that we think of the editor form - eg title first, description, then fn.